src/components/SocialLink.tsx

```diff
@@ -10,12 +10,13 @@ interface SocialMedia {
 
```

Comment:
@hugomarquesdev I believe the social icons still need further styling, the icons don't match with the designs and there are some adjustments that probably will need to be made, i don't think this will pass design review as it is. Maybe it's better to check with design the current layout, so it can be adapted to real use cases like we have in staging.

Plus, all the prismic data in this view as disappeared.

Apart from that, everything seems fine, tested mobile and desktop.

Reply:
@PauloSousapt 

The icons are the same as the website and storybook (impactMarket design system). The phone icon was directly downloaded from figma with paulo soares. I also noticed that they seem different in the app and figma but I don't know why exactly...

I'll check the prismic data.

Reply:
> Plus, all the prismic data in this view as disappeared.

In which view? Community?